### PR TITLE
Updated Changelog.md file as required. 

### PR DESCRIPTION
**Pull request by arshpreetsingh**
Updated changelog.md file.

### CODE REVIEW

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,33 @@
...
 <h3>New features since last release</h3>
 
+* Docker support to build Pennylane for all three interfaces(Tensorflow, Torch and Jax), All 12 External-Pulgins(cirq, qiskit, amazon-braket etc) and Qchem implementaion for GPU+CPU. 
+
+ [(#1372)](https://github.com/PennyLaneAI/pennylane/issues/1372)
+
+Build process using Docker and Makefile works as follows:
+
+To Build basic Pennylane image
+
```
Comment (thisac) [inline .github/CHANGELOG.md:12]:
```suggestion
Building a core PennyLane image:
```

File: .github/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,33 @@
...
+```
+make -f docker/Makefile build-base
+```
+Build a PennyLane image with a specific interface
+
+```
+make -f docker/Makefile build-interface interface-name=tensorflow
+```
+Build a PennyLane image with a plugin
+
```
Comment (thisac) [inline .github/CHANGELOG.md:22]:
```suggestion

Building a PennyLane image with the Qiskit plugin (change `plugin-name` for other plugins):
```

File: .github/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,33 @@
...
+```
+make -f docker/Makefile build-plugin plugin-name=qiskit
+```
+Build a PennyLane-Qchem image
+
```
Comment (thisac) [inline .github/CHANGELOG.md:27]:
```suggestion

Building a PennyLane-Qchem image:
```

**Review by thisac — CHANGES_REQUESTED** [review]
Thanks for adding this. Just a few minor suggestions, and then it should be good to go!

**Comment by arshpreetsingh** [comment]
> Thanks for adding this. Just a few minor suggestions, and then it should be good to go!

Resolved!

**Review by thisac — APPROVED** [review]
Looks good! 💯

**Pull request merged**